### ISSUE TITLE
Wrong default date in tasks screen due to time zone

### ISSUE THREAD

**Issue by jaragunde** [Igalia/phpreport#232]
[Trac import]
Reported by: jaragunde
Original date: Tuesday, 05 May 2015 18:38

When you open the tasks screen, the "today" date seems to be set with regard to the server time. For example, if you are working from UTC-8 in the US and you open the tasks screen in the evening, "today" will already be the next day if your server is in UTC zone.

**Comment by jaragunde** [comment]
[Trac import]
Comment by: jaragunde
Original date: Monday, 29 June 2015 14:06

```
commit 616e1869ca030166801b067c211ee8944a52aa40
Author: Jacobo Aragunde Pérez <jaragunde@igalia.com>
Date:   Mon Jun 29 14:02:24 2015 +0200

    [#232] Get current date from the client instead of the server
    
    In case you open tasks.php with no date specified, a small snipped of
    JS code will be run to get the current date and reload tasks.php with
    the correct date passed as a GET parameter.
```

**Comment by jaragunde** [comment]
The fix to #392 caused a regression on this bug. Reopening.

**Comment by jaragunde** [comment]
```
commit 9abc5adae54c694c6fa9f940548e60e99fa32af1
Author: Jacobo Aragunde Pérez <jaragunde@igalia.com>
Date:   Mon Feb 25 07:56:05 2019 +0100

    [#232][#392] Fix default date based on local timezone for all browsers.
```
